### PR TITLE
Bump version to 3.2.0 and update changelog

### PR DESCRIPTION
**Pull request by mockdeep**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,112 @@
...
+this release possible.
+
+The biggest change in moving towards RMagick 4.0 will be the fact that
+`opacity` is deprecated in favor of `alpha`.
+
+**NOTE: `opacity` is the opposite of `alpha`!!!**
+
+If you are currently passing opacity into methods, you will need to invert the
+value and use the new `alpha:` keyword argument. If you are passing an integer,
+`alpha = 255 - opacity`. An integer `opacity` of 0 is an `alpha` of 255. If you
```
Comment (dlemstra) [inline CHANGELOG.md]: The percentage values have not been inverted so this remark can be removed.

Comment (mockdeep) [reply CHANGELOG.md]: Wait, so do the percentages still represent opacity?

Comment (dlemstra) [reply CHANGELOG.md]: They represent the transparency percentage instead.

Comment (mockdeep) [reply CHANGELOG.md]: Ah, I see. So they've always been transparency? Updated.

Comment (dlemstra) [reply CHANGELOG.md]: Yup!

**Comment by mockdeep** [comment]
Alrighty, released!

**Pull request merged**